packages/geant4reweight/package.py

```diff
@@ -14,6 +14,16 @@ class Geant4reweight(CMakePackage, FnalGithubPackage):
     version_patterns = ["v01_20_00", "01.20.05"]
 
     version("01.20.00", sha256="f8d30f2a1426ee9e100694d4d19d58a7b98af93c8e71ff0a52cb0a1e7a6d3d96")
+    version("01.16.05", sha256="23417293c2bb5663bbe26398c622c08052563febf396fd7513e9c8536687c6e8")
+
+    # experiment versions
+    variant("experiment", default="lar", description="Experiment variants",
+            values=("lar", "nova"), multi=False)
+    conflicts("@01.20.00", when="experiment=nova") # lar flavour
+    conflicts("@01.16.05", when="experiment=lar") # nova flavour
```

Comment:
I suggest expressing this in the positive:

```suggestion
    requires("@01.20.00", when="experiment=lar")
    requires("@01.16.05", when="experiment=nova")
```

Reply:
using `requires` instead of `conflicts` sounds good to me! although in that case, it may make more sense to invert the logic:

```suggestion
    requires("experiment=lar", when="@01.20.00")
    requires("experiment=nova", when="@01.16.05")
```

this would allow more than one version to satisfy each `experiment=XXX` variant, in the event that we need to maintain these divergent variants of the package a while longer. let me know if that works for you, and i'll commit this change. thank you!!

Reply:
That's fine, @vhewes.  Feel free to go ahead with your suggestion.